com.unity.render-pipelines.high-definition/Runtime/PostProcessing/PostProcessSystem.cs

```diff
@@ -70,6 +70,29 @@ private enum SMAAStage
         readonly GraphicsFormat m_LutFormat;
         readonly HableCurve m_HableCurve;
 
+        //Viewport information
+
+        Vector2Int m_FullRes = new Vector2Int(1,1);
+        Vector2Int m_DownsampledRes = new Vector2Int(1,1);
+
+        internal enum ResolutionGroup
+        {
+            Downsampled,
```

Comment:
I'd maybe chose a different name as Downsampled here is downsampled only in the context of dynamic resolution, when no dynamic res is on it might be confusing. 

I don't have yet a better name in mind that is not maga verbose, but will try to think about it a bit :D

Reply:
Maybe the group names are BeforeDRS, AfterDRS?

Reply:
A bit wary on the abbreviation, I think it makes more sense if we flip it on its head and say BeforeDynamicResUpscale  / AfterDynamicResUpscale   

Dunno naming is hard :D

Reply:
I would tend to agree with Francesco here. I think Before/After works better in all cases.

Reply:
this has been fixed!